components/src/core/empty-state/empty-state.tsx

```diff
@@ -109,7 +109,7 @@ export const EmptyState: React.FC<EmptyStateProps> = (props) => {
     }, [icon, normalizeIconSize, getColor, iconColor]);
 
     return (
-        <View style={[defaultStyles.root, styles.root, style]} {...viewProps}>
```

Comment:
Why is this an inline style instead of adding the `padding: 8` to the defaultStyles.root?  If it's an inline styles, doesn't this limit the ability to override it as a user?

Reply:
Yep, these should be dealt with in the defaultStyles.root so the end user can easily override these styles